app/views/partners/children/show.html.erb

```diff
@@ -59,7 +59,11 @@
                     <dd><%= @child.health_insurance %></dd>
 
                     <dt>Item needed:</dt>
-                    <dd><%= current_partner.organization.item_id_to_display_string_map[@child.item_needed_diaperid] %></dd>
+                    <dd>
+                      <%= @child.needed_item_ids.map do |item_id|
+                        current_partner.organization.item_id_to_display_string_map[item_id]
+                      end.join(', ') %>
```

Comment:
Style guide (which doesn't work on ERB) indicates we should use `{ }` curly brackets instead of `do..end` if there's a method that is run on the results of the block.